src/main/java/com/daitangroup/api/services/VehicleService.java

```diff
@@ -0,0 +1,93 @@
+package com.daitangroup.api.services;
+
+import com.daitangroup.api.model.ProductionOrder;
+import com.daitangroup.api.model.Vehicle;
+import com.daitangroup.api.repository.ProductionOrderRepository;
+import com.daitangroup.api.repository.VehicleRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import java.util.Optional;
+
+@Service
+public class VehicleService {
+
+    @Autowired
+    private VehicleRepository vehicleRepository;
+
+    @Autowired
+    private ProductionOrderRepository productionOrderRepository;
+
+    public Optional<Vehicle> findById(long id) {
+        return vehicleRepository.findById(id);
+    }
+
+    public Optional<Vehicle> findByPlate(String plate) {
+        return vehicleRepository.findByPlate(plate);
+    }
+
+    public Iterable<Vehicle> findAll() {
+        return vehicleRepository.findAll();
+    }
+
+    public Optional<ProductionOrder> findPoByVehicleId(long id) {
```

Comment:
Please rename it to represent the entity it's working with.

Since this is a Vehicle service why should we return a ProductionOrder? Maybe this method should be moved to ProductionOrderService